neo/rawio/axonarawio.py

```diff
@@ -629,12 +642,13 @@ def _get_signal_chan_header(self):
         gain_list = self._get_channel_gain()
         offset = 0  # What is the offset?
 
+        first_channel = (active_tetrode_set[0] - 1)*elec_per_tetrode
         sig_channels = []
         for itetr in range(num_active_tetrode):
 
             for ielec in range(elec_per_tetrode):
-                cntr = (itetr * elec_per_tetrode) + ielec
-                ch_name = f"{itetr + 1}{letters[ielec]}"
+                cntr = (itetr * elec_per_tetrode) + ielec + first_channel
+                ch_name = "{}{}".format(itetr + active_tetrode_set[0], letters[ielec])
```

Comment:
```suggestion
                ch_name = f"{itetr + active_tetrode_set[0]}{letters[ielec]}"
```

we try to keep the codebase on f-strings since they are faster and easier to read :)